### PR TITLE
ci(logging): update tests to reflect new 128 bit trace_id standard

### PR DESCRIPTION
**Pull request by zarirhamza**
Updates tests in new logging libraries to reflect new 128 bit trace_id standard. 
The logic remains untouched that 128bit trace_ids continue to only be logged iff `_128_bit_trace_id_logging_enabled` is `True`

## Checklist

- [x] Change(s) are motivated and described in the PR description.
- [x] Testing strategy is described if automated tests are not included in the PR.
- [x] Risk is outlined (performance impact, potential for breakage, maintainability, etc).
- [x] Change is maintainable (easy to change, telemetry, documentation).
- [x] [Library release note guidelines](https://ddtrace.readthedocs.io/en/stable/releasenotes.html) are followed. If no release note is required, add label `changelog/no-changelog`.
- [x] Documentation is included (in-code, generated user docs, [public corp docs](https://github.com/DataDog/documentation/)).
- [x] Backport labels are set (if [applicable](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting))

## Reviewer Checklist

- [x] Title is accurate.
- [x] No unnecessary changes are introduced.
- [x] Description motivates each change.
- [x] Avoids breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
- [x] Testing strategy adequately addresses listed risk(s).
- [x] Change is maintainable (easy to change, telemetry, documentation).
- [x] Release note makes sense to a user of the library.
- [x] Reviewer has explicitly acknowledged and discussed the performance implications of this PR as reported in the benchmarks PR comment.
- [x] Backport labels are set in a manner that is consistent with the [release branch maintenance policy](https://ddtrace.readthedocs.io/en/latest/contributing.html#backporting)
- [x] If this PR touches code that signs or publishes builds or packages, or handles credentials of any kind, I've requested a review from `@DataDog/security-design-and-guidance`.
- [x] This PR doesn't touch any of that.

### CODE REVIEW

File: tests/contrib/logbook/test_logbook_logging.py
Comment (mabdinur) [inline tests/contrib/logbook/test_logbook_logging.py:55]: nit: this can be a fixture that's auto applied to all logbook tests

File: tests/contrib/logbook/test_logbook_logging.py
Comment (mabdinur) [inline tests/contrib/logbook/test_logbook_logging.py:22]: nit: I dont think we need to check if `128_bit_trace_id` is enabled. 

`span._trace_id_64bits == trace_id` if `config._128_bit_trace_id_enabled == False`.

**Review by mabdinur — APPROVED** [review]
It would be nice to have test case for when 128 bit trace id logging is enabled. I think we are only test the default behavior. 

Changes so far look good to me. I don't think we need to block this PR on my current suggestions. Let's unblock ci

**Review by ZStriker19 — APPROVED** [review]

File: tests/contrib/logbook/test_logbook_logging.py
Comment (zarirhamza) [reply tests/contrib/logbook/test_logbook_logging.py:22]: Done

File: tests/contrib/logbook/test_logbook_logging.py
Comment (zarirhamza) [reply tests/contrib/logbook/test_logbook_logging.py:55]: Done

**Comment by zarirhamza** [comment]
> It would be nice to have test case for when 128 bit trace id logging is enabled. I think we are only test the default behavior.
> 
> Changes so far look good to me. I don't think we need to block this PR on my current suggestions. Let's unblock ci

We do have tests that test 128 bit trace Id within the subprocesses. Those tests are explicitly defined with an env var making 128bit trace_ids enabled.

**Pull request merged**
